_scripts/migrations/0027_discussion_shuffle_time.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS discussion_shuffle_time (
+    discussion_id varchar(36) PRIMARY KEY,
+    shuffle_time timestamp with time zone
+);
+
+ALTER TABLE discussion_shuffle_time
+    ADD CONSTRAINT dst_discussion_id_fk_4fa856c56ee0 FOREIGN KEY (discussion_id) REFERENCES discussions(id) MATCH FULL ON DELETE CASCADE;
+
+CREATE INDEX IF NOT EXISTS discussion_shuffle_time_idx on discussion_shuffle_time (shuffle_time);
+
+ALTER TABLE discussions ADD COLUMN shuffle_id int default 0;
```

Comment:
why 0 over null for the default (non-set) value?

Reply:
I want to set it to not null but we cannot until the column exists and we've updated all of the values. I'll follow on with a PR to set it to not null.